jupyterlab_automl/cloudbuild-release.yaml

```diff
@@ -0,0 +1,33 @@
+substitutions:
```

Comment:
Feel free to drop this as well since are moving to GitHub actions for CI/CD.

Reply:
Okay sounds good.